tests/js/spec/views/stream.spec.jsx

```diff
@@ -209,6 +210,7 @@ describe('Stream', function() {
         dataLoading: false,
       });
 
+      expect(wrapper).toMatchSnapshot();
```

Comment:
Does this snapshot add value over the assertion below?

Reply:
This one less so, will remove then